waku/v2/node/rest/json_serdes.nim

```diff
@@ -0,0 +1,57 @@
+import std/typetraits
+import stew/results,
+       serialization,
+       json_serialization,
+       json_serialization/std/[options, net, sets],
+       chronicles
+
+logScope: topics = "api_rest_json_serdes"
```

Comment:
Perhaps we should create a general refactoring issue where we fix the `logScope` for all Waku modules to something consistent. I have no specific preference and like this `_underscore_` convention. Other modules, however, are all over the place, with `dot.notation`, `spa ces`, etc. sometimes with `waku.` prefix and sometimes without any prefix. Preferably we want to prefix the scope for related modules so their logs are easily recognisable in a noisy file, as you've done here with `api_`. 👍

Reply:
We have this issue: https://github.com/status-im/nwaku/issues/983

From my POC, that one looks to me like the perfect moment to agree and apply the convention.